templates/providerstoragecluster.go

```diff
@@ -31,8 +31,46 @@ import (
 
 const (
 	ProviderOSDSizeInTiB = 4
+	osdNodeLabel         = "node.ocs.openshift.io/osd"
```

Comment:
- due to OCM limitations, once set changing this field isn't possible
- for reviewers, pls provide an ack whether this is enough or some more info is required in the key, i.e.,
```
osdNodeLabelKey = "node.ocs.openshift.io/reserved-for"
osdNodeLabelValue = "ceph-osd"
```
- and the same will be changed in this file